server/metrics/metrics.go

```diff
@@ -202,6 +204,15 @@ func NewMetrics(info InstanceInfo) Metrics {
 	})
 	m.registry.MustRegister(m.linkedChannelsTotal)
 
+	m.upstreamUsersTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemApp,
+		Name:        "upstream_users_total",
```

Comment:
constant maybe ?

Reply:
this is not used anywhere else. I don't think we need constant for this currently

Reply:
If I want to see all these names in 1 place all these metrics like in this case upstream_users_total, Can I do that ?

Reply:
@ayusht2810

Reply:
we already have interface present, from which we can check all the metrics present in the file